src/javascripts/components/code-blocks.mjs

```diff
@@ -0,0 +1,50 @@
+/**
+ * Responsive behaviour for code blocks
+ */
+class CodeBlocks {
+  /**
+   * @param {NodeListOf<Element>} $elements
+   */
+  constructor($elements) {
+    if (!$elements.length) {
+      return this
+    }
+
+    // Check if we can use Resize Observers, backing out if not
+    if (!('ResizeObserver' in window)) {
```

Comment:
🙌🏻 [Good catch!](https://browsersl.ist/#q=fully+supports+es6-module+and+not+supports+resizeobserver)